src/main/java/org/openhab/binding/openocean/internal/eep/EEPType.java

```diff
@@ -322,7 +330,10 @@
             THING_TYPE_MEASUREMENTSWITCH, CHANNEL_GENERAL_SWITCHINGA, CHANNEL_GENERAL_SWITCHINGB,
             CHANNELTYPE_REPEATERMODE, CHANNEL_RECEIVINGSTATE),
     SwitchWithEnergyMeasurment_12(RORG.VLD, 0x01, 0x12, true, D2_01_12.class, THING_TYPE_MEASUREMENTSWITCH,
-            CHANNEL_GENERAL_SWITCHINGA, CHANNEL_GENERAL_SWITCHINGB, CHANNEL_RECEIVINGSTATE);
+            CHANNEL_GENERAL_SWITCHINGA, CHANNEL_GENERAL_SWITCHINGB, CHANNEL_RECEIVINGSTATE),
+
+    SwitchWithEnergyMeasurment_A5(RORG._4BS, 0x12, 0x01, false, A5_12_01.class, THING_TYPE_MEASUREMENTSWITCH,
```

Comment:
That's fine, you can use the new thing when you merge it. For me it will not be a seperate thing, but I plan to combine this EEP with the existing D2 measurementSwitch.